readthedocs/vcs_support/backends/git.py

```diff
@@ -150,6 +150,28 @@ def use_shallow_clone(self):
         from readthedocs.projects.models import Feature
         return not self.project.has_feature(Feature.DONT_SHALLOW_CLONE)
 
+    @staticmethod
+    def _is_trigger_integration_gitlab(project):
+        # If there is only one integration associated with the project, we
+        # can unambiguously conclude that the external build was triggered by
+        # a merge request.
+        #
+        # https://github.com/readthedocs/readthedocs.org/issues/9464
+
+        # To avoid error like the following, the value corresponding to
+        # Integration.GITLAB_WEBHOOK is hardcoded.
+        #
+        #   ImportError: cannot import name 'Project' from partially initialized module
+        #   'readthedocs.projects.models' (most likely due to a circular import)
+        _gitlab_webhook = "gitlab_webhook"  # Integration.GITLAB_WEBHOOK
```

Comment:
If it's a circular import issue, we could import this constant from inside this static method to solve this problem.

Reply:
Good point, I will update accordingly